rpcs3/Emu/System.cpp

```diff
@@ -1416,17 +1416,27 @@ void Emulator::Run(bool start_playtime)
 	}
 }
 
-bool Emulator::Pause()
+bool Emulator::Pause(bool no_resume)
 {
 	const u64 start = get_system_time();
 
+	const system_state pause_state = no_resume ? system_state::fatal : system_state::paused;
+
 	// Try to pause
-	if (!m_state.compare_and_swap_test(system_state::running, system_state::paused))
+	if (!m_state.compare_and_swap_test(system_state::running, pause_state))
 	{
-		if (!m_state.compare_and_swap_test(system_state::ready, system_state::paused))
+		if (!no_resume)
```

Comment:
this doesn't make any sense.
this means you can never go from ready to paused

Reply:
I don't understand why anyone want that. Please elaborate.

Reply:
nevermind then.

But i found another issue (maybe unrelated to Pause).

Start games into Ready state but don't press play.
If you now Press stop, the "Emulator took too long"